week-4/InClass/C-more-complex-objects/exercise-3.js

```diff
@@ -32,17 +32,18 @@ let parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
+  return `${house.currentOwner.firstName} ${house.currentOwner.lastName}`
 
 }
 
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return [`${house1.currentOwner.email} , ${house2.currentOwner.email}`]
 }
 
 // returns the address for the cheapest house out of the two
 function getCheapestAddress(house1, house2) {
-
+  return house1.price<house2.price? house1.address:house2.address

```

Comment:
Good use of the ternary operator, but remember your spacing between the syntax. It keeps things more readable.